opentelemetry-etw-logs/README.md

```diff
@@ -21,12 +21,26 @@ captured by agents running locally and listening for specific ETW events.
 [![Slack](https://img.shields.io/badge/slack-@cncf/otel/rust-brightgreen.svg?logo=slack)](https://cloud-native.slack.com/archives/C03GDP0H023)
 
 ## Viewing ETW Logs
-
 Logs exported to ETW can be viewed using tools like `logman`, `perfview` etc.
-// TODO - add instructions.
 
+#### Instructions using Powershell:
```

Comment:
i dont think we need to duplicate the instructions from https://github.com/open-telemetry/opentelemetry-rust-contrib/blob/main/opentelemetry-etw-logs/examples/basic.rs#L7 to readme file.

It'd be better to use https://github.com/open-telemetry/opentelemetry-rust-contrib/blob/main/opentelemetry-etw-logs/src/lib.rs#L1 for documentation so it show up docs.rs .